src/utils/index.ts

```diff
@@ -132,7 +132,12 @@ export function readPackageJson (
   }
 }
 
-export function resolveAliases (aliases: Record<string, string>) {
+export function resolveAliases (_aliases: Record<string, string>) {
+  // Sort aliases from specific to general (ie. fs/promises before fs)
+  const aliases = Object.fromEntries(Object.entries(_aliases).sort(([a], [b]) => {
+    return b.split('/').length - a.split('/').length
```

Comment:
Maybe fallback to last subpath length if both have same segments?

Reply:
Why?

Reply:
I mean if there is `foo/bar` `foo/bar-123`, they have same length of 2. Then we can compare them by their length. (In case alias is not a file but prefix)